### PR TITLE
Refactor removing model from graphical editor

### PR DESCRIPTION
**Pull request by kmilena98**
We have removed the model from the graphical editor and created DTO objects for communication with the backend. We have a Trello card for this story but somehow it doesn't have the number, that is why the name of the branch is not in its proper form.

### CODE REVIEW

File: HealthcareSystem/GraphicalEditorServer/Mappers/pubilc.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace GraphicalEditorServer.Mappers
+{
+    internal class pubilc
+    {
+    }
+}
```
Comment (vlaksi) [inline HealthcareSystem/GraphicalEditorServer/Mappers/pubilc.cs]: For what is this used?

Comment (kmilena98) [reply HealthcareSystem/GraphicalEditorServer/Mappers/pubilc.cs]: I guess it has been added by mistake, I have removed it from the code.

**Comment by vlaksi** [comment]
Is there some specific reason why you use static methods [there](https://github.com/PSW-2020-ORG4/HealthcareSystem/pull/160/files#diff-fb6d80b1902833aef6af58d72c5c5fe402e8e65436ced03a0677751e712f1006) and [there](https://github.com/PSW-2020-ORG4/HealthcareSystem/pull/160/files#diff-6f63cbdf1f11a44283d5550245d00e151d64018fb0a755d4d48aa09ff85e1d53)

**Comment by kmilena98** [comment]
> Is there some specific reason why you use static methods [there](https://github.com/PSW-2020-ORG4/HealthcareSystem/pull/160/files#diff-fb6d80b1902833aef6af58d72c5c5fe402e8e65436ced03a0677751e712f1006) and [there](https://github.com/PSW-2020-ORG4/HealthcareSystem/pull/160/files#diff-6f63cbdf1f11a44283d5550245d00e151d64018fb0a755d4d48aa09ff85e1d53)

We made all method in mappers static so we can call it directly from the class without making a mapper object.

**Review by vlaksi — DISMISSED** [review]

**Comment by vlaksi** [comment]
> > Is there some specific reason why you use static methods [there](https://github.com/PSW-2020-ORG4/HealthcareSystem/pull/160/files#diff-fb6d80b1902833aef6af58d72c5c5fe402e8e65436ced03a0677751e712f1006) and [there](https://github.com/PSW-2020-ORG4/HealthcareSystem/pull/160/files#diff-6f63cbdf1f11a44283d5550245d00e151d64018fb0a755d4d48aa09ff85e1d53)
> 
> We made all methods in mapper static so we can call it directly from the class without making a mapper object.

For me personally, it's better to avoid static methods whenever you can, but if you need to use it like this. It's okay for me.

File: HealthcareSystemFrontends/GraphicalEditor/Service/GenericHTTPService.cs [hunk 2]
```diff
@@ -13,29 +12,20 @@ public class GenericHTTPService
...
-            var request = new RestRequest("api/" + requestURL);
-            request.AddJsonBody(JSONContent);
-            IRestResponse response = client.Post(request);
-            return response;
-        }
-
-        public List<T> HTTPGetRequestWithObjectAsParqm<T>(string requestURL, object objectParam)
+        }       
+       
+        public List<T> HTTPGetRequestWithObjectAsParam<T>(string requestURL, object objectParam)
```
Comment (ogikuzma) [inline HealthcareSystemFrontends/GraphicalEditor/Service/GenericHTTPService.cs:19]: This method has GET in the name, but POST in the request. It should be changed.

**Review by ogikuzma — DISMISSED** [review]

**Comment by kmilena98** [comment]
[(5) Refactoring code from previous sprints and improving design of the system](https://trello.com/c/tGV6jwRF/48-5-refactoring-code-from-previous-sprints-and-improving-design-of-the-system)

**Review by vlaksi — APPROVED** [review]

**Review by VasilijeBursac — APPROVED** [review]

**Pull request merged**